handbook/communication/code_of_conduct.md

```diff
@@ -0,0 +1,180 @@
+
+Keep the following things in mind:
+
+-   When representing Sourcegraph, be mindful that your words are a reflection on Sourcegraph and the many individuals who make up the team.
+    -   Those in management roles should be especially mindful of their personal beliefs and biases when acting on behalf of the company.
+-   Appreciate uniqueness and do not treat individual opinions as representative of the opinion of the larger group.
+-   Engage others to ensure their voices are heard. Constructive dissent should be addressed directly.
+-   Public polls and private ballots often do not yield the same result. Keep this in mind when gauging consensus.
+-   Handle interpersonal conflict with professionalism and integrity. If there is strong disagreement, feedback should be given directly and clearly, without heat, or implicit denigration.
+-   You will work with others who have personal opinions or beliefs that differ from your own. Strive to overcome these differences and find common cause.
+-   Be aware of power differentials. If you have the ability to threaten someone’s job, your criticism may be taken more harshly than you intend.
+
+This code is not exhaustive or complete. It serves to capture our common understanding of a productive, collaborative environment. We expect the code to be followed in spirit as much as in the letter.
+
+
+## Our responsibilities
+
+Leadership is responsible for clarifying the standards of acceptable behavior and is expected to take appropriate and fair corrective action in response to any instances of unacceptable behavior.
+
+Leadership has the right and responsibility to remove, edit, or reject comments, commits, code, wiki edits, issues, and other contributions that are not aligned to this Code of Conduct, or to ban temporarily or permanently any contributor for other behaviors that they deem inappropriate, threatening, offensive, or harmful.
+
+
+## Scope
+
+This code of conduct applies to any individual or community member that is representing the project or its community within both project spaces and public spaces, including (but not limited to) social media, online or offline events, and email or Slack exchanges.
+
+Note: Some Sourcegraph-managed communities have codes of conduct that pre-date this document and issue resolution process. While communities are not required to change their code, they are expected to use the resolution process outlined here. The review team will coordinate with the communities involved to address any concerns.
+
+
+## Enforcement
+
+We encourage all communities, teams, employees, and team members to resolve issues on their own whenever possible. This builds a broader and deeper understanding and ultimately a healthier interaction.
+
+Instances of abusive, harassing, or otherwise unacceptable behavior may be reported by contacting conduct@sourcegraph.com. The conduct team is obligated to maintain confidentiality of the person reporting the incident at all times.
```

Comment:
```suggestion
Instances of abusive, harassing, or otherwise unacceptable behavior may be reported by contacting [conduct@sourcegraph.com](mailto:conduct@sourcegraph.com). The conduct team is obligated to maintain the confidentiality of the person reporting the incident at all times.
```